Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -178,6 +178,18 @@ def keg_contain?(string, keg, ignores)
           offset, match = str.split(" ", 2)
           next if linked_libraries.include? match # Don't bother reporting a string if it was found by otool
 
+          # Do not report matches to directories that do not exist.
+          dir = Pathname.new(File.dirname(match))
+          next unless dir.directory?
+
+          # Do not report matches to build dependencies.
+          begin
+            keg_name = Keg.for(dir).name
+            next unless @runtime_deps.include? keg_name
```

Comment:
Could this be passed through as an argument (to `keg_contain?`) instead? More obvious to me than relying on a instance variable declared elsewhere.

Reply:
Done.